ui/packages/app/src/components/UploadButton/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface FileData {
```

Comment:
You've defined this twice, can you reuse and import from the other? Ideally, we should merge all the definitions of interface FileData into `ui/packages/app/src/components/types.ts` instead of reimplenting over and over, if you can do that that'd be great.